### PR TITLE
wip

### PR DESCRIPTION
**Pull request by GitBluub**

### CODE REVIEW

**Review by Octopus773 — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -20,7 +20,5 @@ API_KEYS=SCOROTEST,ROBOTO,SCORO
 API_KEY_SCORO_TEST=SCOROTEST
 API_KEY_ROBOT=ROBOTO
 API_KEY_SCORO=SCORO
-MEILI_HTTP_ADDR="http://meilisearch:7700"
-MEILI_MASTER_KEY="ghvjkgisbgkbgskegblfqbgjkebbhgwkjfb"
```
Comment (zoriya) [inline .env.example:24]: Pretty sure you need the master key, meilisearch does not run in prod mode without.

File: docker-compose.yml [hunk 2]
```diff
@@ -62,10 +62,19 @@ services:
...
     volumes:
       - meilisearch:/meili_data
     env_file:
       - .env
+    healthcheck:
+      test:
+          - CMD
+          - wget
+          - '--no-verbose'
+          - '--spider'
```
Comment (zoriya) [inline docker-compose.yml:77]: what?

File: .env.example (hunk 1 above)
Comment (GitBluub) [reply .env.example:24]: We can add it for prod, not needed in the ci I think

File: docker-compose.yml (hunk 2 above)
Comment (GitBluub) [reply docker-compose.yml:77]: healthcheck from demo project by meilisearch

Comment (zoriya) [reply docker-compose.yml:77]: nice, no clue what --spider does

**Review by zoriya — APPROVED** [review]

**Pull request merged**